modules/ROOT/pages/apikit-4-forgraphql.adoc

```diff
@@ -3,13 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-GraphQL is a data query and manipulation language that enables API consumers to ask for the data that is only relevant to them. With GraphQL, you can develop GraphQL APIs, which are type-system specifications written in a GraphQL-schema language. 
+GraphQL is a data query and manipulation language that enables API consumers to ask for the data that is only relevant to them. The APIkit for GraphQL module supports the implementation of GraphQL services. These services use connectors to map GraphQL APIs to the existing backends where source data is stored.
 
-The APIkit for GraphQL module supports the implementation of GraphQL services. These services use connectors to map GraphQL APIs to the existing backends where source data is stored.
+APIkit scaffolder enables to quickly implement projects by generating the initial skeleton of the application at design time based on the GraphQL schema. The application design maps queries, mutations, and types in your API to the Mule flows so you only specify how your data sources fulfill a particular data type.
```

Comment:
I would make a link/mention the ACB documentation section were this is explained

Reply:
I agree! I'll get in contact with Fer to add a link to ACB docs. Thanks!